alert-policies/mexn/MongoDBConnectionsCount.yml

```diff
@@ -0,0 +1,41 @@
+# Name of the alert
+name: MongoDB Collection Count
+
+# Description and details
+description: |+
+  This alert is triggered when the created MongoDB connections are higher than 2000 for 5 minutes.
```

Comment:
Please consider rephrasing to:
This alert is triggered when the there are more than 2000 MongoDB connections created within a 5 minute period.